lisa/tests/scheduler/eas_behaviour.py

```diff
@@ -59,35 +59,41 @@ def nrg_model(self):
         return self.plat_info['nrg-model']
 
     @classmethod
-    def get_big_duty_cycle(cls, plat_info, load):
+    def get_big_duty_cycle(cls, plat_info, utilization_pct=None):
         """
-        For load 0 or negative it returns the smallest duty-cycle percentage
-        guaranteeing the task will be placed on the biggest CPUs of the system.
-        This allows minimizing thermal issues while ensuring the expected
-        placement. Otherwise it return the duty-cycle percentage equivalent to
-        load on big CPUs.
+        Returns a duty cycle for :class:`lisa.wlgen.rta.Periodic` that will
+        guarantee placement on a big CPU.
+
+        :param utilization_pct: If ``None``, the duty cycle will be chosen so
+            that the task will not fit on the second to biggest CPUs in the
+            system, thereby forcing up-migration while minimizing the thermal
+            impact. If a number is passed, the value will be used as a
+            percentage of utilization of a big CPU, and will be converted to a
+            duty cycle.
+        :type utilization_pct: int or None
         """
         # If a negative/zero load is passed we return the duty-cycle equivalent
```

Comment:
You should remove this as now there is enough detail in the description! Thanks!